mobile-widgets/qml/DivePlannerSetup.qml

```diff
@@ -23,8 +23,8 @@ Kirigami.ScrollablePage {
 			spinBestmixend.value = Planner.bestmixend
 		}
 		onVolumeChanged: {
-			spinBottomsac.value = Planner.bottomsac
-			spinDecosac.value = Planner.decosac
+			spinBottomsac.value = Planner.bottomsac * ((PrefUnits.volume === Enums.LITER) ? 1 : 100)
+			spinDecosac.value = Planner.decosac * ((PrefUnits.volume === Enums.LITER) ? 1 : 100)
```

Comment:
Ugh. So we are getting adjusted values back, but we still end up having to have conditional code here for scaling? That's just strange and ugly. This should return a metric value and we should have a JS function that converts it to and from the display value.

Reply:
I agree this is ugly, and needs to be corrected !! This is a case where a shared conversion does not help anything at all.

but for the general argument the case for making plannershared.* was to share the conversions between desktop and mobile. If your preference have changed, then we double quite a number of conversions, is that what is wanted ?